Sources/Basics/FileSystem+Extensions.swift

```diff
@@ -8,7 +8,6 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
  */
 
-import TSCBasic
```

Comment:
I don't think we should be removing the `TSCBasic` imports from all files since it currently has plenty of other types besides collections. We may want to remove the exports file prior not having `TSCBasic` anymore and then we have to put all those imports back again. If we have any files that only import `TSCBasic` for collections, we can remove the import from those of course.

Reply:
what I did here was to replace the direct import on `TSCBasic` with an in import of `Basics` which currently re-exported by `TSCBasics`, the idea is that we can evolve `Basics` over time until we can eventually eliminate the need in `TSCBasic`

Reply:
I kinda feel like that could make it harder to remove `TSCBasic`? Since now every client of `Basics` will see the API of `TSCBasic` for an extended period of time. IMO, it would be good to get rid of the re-export quickly.

Reply:
is https://github.com/apple/swift-package-manager/pull/3595 better?

Reply:
Yah, I think I prefer that approach.